PhpGedView/languages/CHANGELOG_configure_help.en.txt
PHPGEDVIEW LANGUAGE FILE CHANGELOG

# $Id: CHANGELOG_configure_help.en.txt 6977 2010-08-25 01:06:08Z canajun2eh $

For information on changes prior to the release of PGV 4.2.0 (16 Nov. 2008), please 
refer to the files in ZIP file "languages/CHANGELOG_bef.20081117.zip"

Use this file to follow the changes made to the English language files
between different versions of PhpGedView.  There are eight separate CHANGELOG files,
one to correspond to each file that together make up the entire set of English text.

The CHANGELOG files are:
		CHANGELOG_admin.en.txt
		CHANGELOG_configure_help.en.txt
		CHANGELOG_countries.en.txt
		CHANGELOG_editor.en.txt
		CHANGELOG_facts.en.txt
		CHANGELOG_faqlist.en.txt
		CHANGELOG_help_text.en.txt
		CHANGELOG_lang.en.txt

+ $pgv_lang["text"] / means added
- $pgv_lang["text"] / deleted
* $pgv_lang["text"] / translation was changed
# $pgv_lang["text"] / keyword was changed

Additional codes, to help Translators, follow the date as follows:
	A - Added or moved from another file
	C - Capitalization  (doesn't affect translation)
	D - Deleted
	f - Minor format change
	F - Major format change
	p - Punctuation change  (doesn't affect translation)
	M - Moved to another file
	R - keyword was renamed (no translation changes required)
	S - Spelling change		(doesn't affect translation)
	w - Minor wording change
	W - Major wording change or completely new text

The above list of codes may be amended as new situations arise.

VERSION 4.2.4

?$pgv_lang["whatever"]								dd Monthname, year		Change	codes and comments
----------------------------------------------------------------------------------------
+$pgv_lang["croatian"]								23 Mar 2010				A
*$pgv_lang["add_faq_pgvLangRefs"]					07 Feb 2010				W	Added example of a reference to a "PGV_..." constant
*$pgv_lang["um_bu_logs"]							24 Jan 2010				W 
*$pgv_lang["USE_SILHOUETTE_help"]					05 Jan 2010				w added missing '>'
*$pgv_lang["USE_SILHOUETTE_help"]					03 Jan 2010				W
+$pgv_lang["USE_SILHOUETTE"]						02 Jan 2010				A
+$pgv_lang["USE_SILHOUETTE_help"]					02 Jan 2010				A


VERSION 4.2.3

?$pgv_lang["whatever"]								dd Monthname, year		Change codes and comments
-----------------------------------------------------------------------------------------
+$pgv_lang["lang_name_english (uk)"]				04 Nov 2009				A
+$pgv_lang["lang_name_english (us)"]				04 Nov 2009				A
-$pgv_lang["ALLOW_REMEMBER_ME_help"]				30 Sep 2009				D
-$pgv_lang["META_SURNAME_KEYWORDS"]					27 Sep 2009				D
-$pgv_lang["META_SURNAME_KEYWORDS_help"]			27 Sep 2009				D	
*$pgv_lang["PGV_SMTP_HOST_help"]					10 Aug 2009				W
*$pgv_lang["PGV_SMTP_SSL_help"]						10 Aug 2009				W


VERSION 4.2.2

?$pgv_lang["whatever"]								dd Monthname, year		Change codes and comments
-----------------------------------------------------------------------------------------
*$pgv_lang["DB_UTF8_COLLATION_help"]				24 Jun 2009				w
-$pgv_lang["DBPERSIST"]								04 Jun 2009				D
-$pgv_lang["DBPERSIST_help"]			 			04 Jun 2009				D
*$pgv_lang["DBTYPE_help								16 May 2009				Sw
*$pgv_lang["DBTYPE_help								16 May 2009				W


VERSION 4.2.1

?$pgv_lang["whatever"]								dd Monthname, year		Change codes and comments
-----------------------------------------------------------------------------------------
*$pgv_lang["ENABLE_AUTOCOMPLETE"]					27 Mar 2009				A
*$pgv_lang["ENABLE_AUTOCOMPLETE_help"]				27 Mar 2009				A
*$pgv_lang["USE_THUMBS_MAIN_help"]					22 Mar 2009				W
*$pgv_lang["PGV_SMTP_SSL_help"]						15 Feb 2009				W
+$pgv_lang["PGV_SMTP_SSL_help"]						15 Feb 2009				A
*$pgv_lang["USE_THUMBS_MAIN_help"]					13 Feb 2009				w
+$pgv_lang["lang_name_serbian-la"]					12 Feb 2009				A


VERSION 4.2.0

?$pgv_lang["whatever"]								dd Monthname, year		Change codes and comments
-----------------------------------------------------------------------------------------
*$pgv_lang["PAGE_AFTER_LOGIN_help"]					30 Jan 2009				w
+$pgv_lang["PAGE_AFTER_LOGIN"]						29 Jan 2009				A
+$pgv_lang["PAGE_AFTER_LOGIN_help"]					29 Jan 2009				A
*$pgv_lang["PREFER_LEVEL2_SOURCES"]					26 Jan 2009				W
*$pgv_lang["USE_QUICK_UPDATE_help"]					26 Jan 2009				W
*$pgv_lang["USE_GEONAMES_help"]						25 Jan 2009				W
*$pgv_lang["USE_GEONAMES"]							24 Jan 2009				W
*$pgv_lang["USE_GEONAMES_help"]						24 Jan 2009				W
+$pgv_lang["USE_GEONAMES"]							24 Jan 2009				A
+$pgv_lang["USE_GEONAMES_help"]						24 Jan 2009				A
+$pgv_lang["lang_name_indonesian"]					24 Jan 2009				A
#$pgv_lang["more_config_hjaelp"]					21 Jan 2009				R	"more_config_hjaelp" --> "more_help_advice"
*$pgv_lang["general_privacy_help"]					21 Jan 2009				w	"more_config_hjaelp" --> "more_help_advice"
*$pgv_lang["PGV_SMTP_FROM_NAME_help"]				15 Jan 2009				W
+$pgv_lang["PGV_SMTP_FROM_NAME_help"]				15 Jan 2009				A
+$pgv_lang["lang_name_slovenian"]					7 Jan 2009				A
*$pgv_lang["PGV_SMTP_ACTIVE_help"]					22 Dec 2008				W
*$pgv_lang["PGV_SMTP_HOST_help"]					22 Dec 2008				W
*$pgv_lang["PGV_SMTP_HELO_help"]					22 Dec 2008				W
*$pgv_lang["PGV_SMTP_PORT_help"]					22 Dec 2008				W
*$pgv_lang["PGV_SMTP_AUTH_help"]					22 Dec 2008				W
*$pgv_lang["PGV_SMTP_AUTH_USER_help"]				22 Dec 2008				W
*$pgv_lang["PGV_SMTP_AUTH_PASS_help"]				22 Dec 2008				W
+$pgv_lang["PGV_SMTP_ACTIVE_help"]					22 Dec 2008				A
+$pgv_lang["PGV_SMTP_HOST_help"]					22 Dec 2008				A
+$pgv_lang["PGV_SMTP_HELO_help"]					22 Dec 2008				A
+$pgv_lang["PGV_SMTP_PORT_help"]					22 Dec 2008				A
+$pgv_lang["PGV_SMTP_AUTH_help"]					22 Dec 2008				A
+$pgv_lang["PGV_SMTP_AUTH_USER_help"]				22 Dec 2008				A
+$pgv_lang["PGV_SMTP_AUTH_PASS_help"]				22 Dec 2008				A
-$pgv_lang["USE_RTL_FUNCTIONS"]						21 Dec 2008				D
-$pgv_lang["USE_RTL_FUNCTIONS_help"]				21 Dec 2008				D
*$pgv_lang["person_privacy_help"]					30 Nov 2008				w	"ID 101" --> "ID I101"
*$pgv_lang["user_privacy_help"]						30 Nov 2008				w	"ID 100" --> "ID I100"
*$pgv_lang["edit_lang_utility_desc"]				28 Nov 2008				w	four --> nine
-$pgv_lang["media_import_help"]						28 Nov 2008				D
*$pgv_lang["compare_lang_utility_desc"]				28 Nov 2008				W	change wording about CHANGELOG files
*$pgv_lang["ENABLE_RSS_help"]						26 Nov 2008				w	Correct URL to Wiki article on RSS
*$pgv_lang["RSS_FORMAT_help"]						26 Nov 2008				w	Correct URL to Wiki article on RSS
*$pgv_lang["EDIT_AUTOCLOSE_help"]					24 Nov 2008				S
*$pgv_lang["DB_UTF8_COLLATION"]						24 Nov 2008				S	"UTF8" --> "UTF-8"
*$pgv_lang["DB_UTF8_COLLATION_help"]				24 Nov 2008				SW
*$pgv_lang["SURNAME_TRADITION_help"]				22 Nov 2008				W
*$pgv_lang["multi_letter_equiv_help"]				22 Nov 2008				CSf
*$pgv_lang["SURNAME_TRADITION_help"]				22 Nov 2008				W
+$pgv_lang["multi_letter_equiv"]					21 Nov 2008				A
+$pgv_lang["multi_letter_equiv_help"]				21 Nov 2008				A
+$pgv_lang["DB_UTF8_COLLATION"]						18 Nov 2008				A
+$pgv_lang["DB_UTF8_COLLATION_help"]				18 Nov 2008				A
+$pgv_lang["collation"]								17 Nov 2008				A
+$pgv_lang["collation_help"]						17 Nov 2008				A


PhpGedView/config.php
<?php
/**
 * Main configuration file required by all other files in PGV
 *
 * The variables in this file are the main configuration variable for the site
 * Gedcom specific configuration variables are stored in the config_gedcom.php file.
 * Site administrators may edit these settings online through the install.php file.
 *
 * phpGedView: Genealogy Viewer
 * Copyright (C) 2002 to 2009  PGV Development Team.  All rights reserved.
 *
 * This program is free software; you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation; either version 2 of the License, or
 * (at your option) any later version.
 *
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with this program; if not, write to the Free Software
 * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 *
 * @package PhpGedView
 * @subpackage Admin
 * @see install.php
 * @version $Id: config.dist 6692 2010-01-07 23:26:51Z fisharebest $
 */

if (preg_match('/\Wconfig.php/', $_SERVER['SCRIPT_NAME'])>0) {
	print 'Got your hand caught in the cookie jar.';
	exit;
}

$DBTYPE='mysql'; //-- type of database to connect when using the PHP/PDO
$DBHOST='localhost'; //-- Host where MySQL database is kept
$DBPORT=''; //-- Database port, leave blank for default
$DBUSER='root'; //-- MySQL database User Name
$DBPASS='mcwskw1'; //-- MySQL database User Password
$DBNAME='db1098262_ancestors'; //-- The MySQL database name where you want PHPGedView to build its tables
$DB_UTF8_COLLATION=false; //-- Use the database to sort/collation UTF8 text
$TBLPREFIX=''; //-- prefix to include on table names
$INDEX_DIRECTORY = "./index/";					//-- Readable and Writeable Directory to store index files (include the trailing "/")
$AUTHENTICATION_MODULE = "authentication.php";	//-- File that contains authentication functions
$PGV_STORE_MESSAGES = true;						//-- allow messages sent to users to be stored in the PGV system
$PGV_SIMPLE_MAIL = true;						//-- allow admins to set this so that they can override the name <emailaddress> combination in the emails
$USE_REGISTRATION_MODULE = true;				//-- turn on the user self registration module
$REQUIRE_ADMIN_AUTH_REGISTRATION = true;		//-- require an admin user to authorize a new registration before a user can login
$ALLOW_USER_THEMES = true;						//-- Allow user to set their own theme
$ALLOW_CHANGE_GEDCOM = true;					//-- A true value will provide a link in the footer to allow users to change the gedcom they are viewing
$LOGFILE_CREATE = "monthly";					//-- set how often new log files are created, "none" turns logs off, "daily", "weekly", "monthly", "yearly"
$LOG_LANG_ERROR = false;						//-- Set if non-existing language variables should be written to a logfile
$PGV_SESSION_SAVE_PATH = "";					//-- Path to save PHP session Files -- DO NOT MODIFY unless you know what you are doing
												//-- leaving it blank will use the default path for your php configuration as found in php.ini
$PGV_SESSION_TIME = "7200";						//-- number of seconds to wait before an inactive session times out
$SERVER_URL=''; //-- the URL used to access this server
$LOGIN_URL = "";								//-- the URL to use to go to the login page, use this value if you want to redirect to a different site when users login, useful for switching from http to https
$MAX_VIEWS = "20";								//-- the maximum number of page views per xx seconds per session
$MAX_VIEW_TIME = "1";							//-- the number of seconds in which the maximum number of views must not be reached
$PGV_MEMORY_LIMIT = "128M";						//-- the maximum amount of memory that PGV should be allowed to consume
$COMMIT_COMMAND = "";							//-- Choices are empty string, cvs or svn
$PGV_SMTP_ACTIVE = false;						//-- Enable to use of SMTP to send external emails
$PGV_SMTP_HOST = "";							//-- The domain name of the SMTP smarthost
$PGV_SMTP_HELO = "";							//-- The local SMTP domain name
$PGV_SMTP_PORT = "25";							//-- The port number talk to the SMTP smarthost
$PGV_SMTP_AUTH = false;							//-- Enable the use of SMTP authorization
$PGV_SMTP_AUTH_USER = "";						//-- User name for SMTP authorization
$PGV_SMTP_AUTH_PASS = "";						//-- Password for SMTP authorization
$PGV_SMTP_SSL = false;							//-- Use SSL for SMTP authorization
$PGV_SMTP_FROM_NAME = "";						//-- Sender name

$CONFIGURED=true; 
require_once './includes/session.php';
